src/openforms/forms/api/serializers.py

```diff
@@ -153,6 +153,10 @@ class FormSerializer(serializers.ModelSerializer):
         default=list,
     )
     login_options = LoginOptionsReadOnlyField()
+    auto_login_authentication_backend = serializers.CharField(
```

Comment:
I am missing validation, tests and documentation that this value must be one of the _selected_ options in the `authentication_backends` field.

Reply:
I removed the `AuthenticationBackendChoiceField` and added the validation you mentioned to the serializer